spec/brainstem/api_docs/formatters/open_api_specification/helper_spec.rb

```diff
@@ -0,0 +1,210 @@
+require 'spec_helper'
+require 'brainstem/api_docs/formatters/open_api_specification/helper'
+
+module Brainstem
+  module ApiDocs
+    module Formatters
+      module OpenApiSpecification
+        describe "Helper" do
```

Comment:
Any reason not to use the constant `Helper` instead of the string here?

Reply:
Copy Pasta :) I'll fix it